src/Tasks/Movie/Main.java

```diff
@@ -0,0 +1,196 @@
+package Tasks.Movie;
+
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Main {
+
+    static Scanner sc = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        MovieApp[] movies = new MovieApp[3];
```

Comment:
in these case, we should send this array to the methods each time, instead you can declare it in class level, and pls initialize these array with 0, we don't want to store null values throughout the flow